doc/usage.md

```diff
@@ -0,0 +1,110 @@
+Usage
+=====
+
+You can see [in this youtube video](https://youtu.be/sXyFa_r1nxA) a very
+short session on how to use RISE to create and run a slideshow.
+
+Let us emphasize the key points here.
+
+## Creating a slideshow
+
+In the notebook menu, the *"View"* option contains a *"Cell Toolbar"*
+sub-menu that gives you access to the metadata for each cell. If you
+select the *"Slideshow"* preset, you will see in the right corner of each
+cell a little box where you can select the cell type.
+
+You can choose between the following types:
+
+* **slide**: this cell is the beginning of a new slide
+
+* **subslide**: this cell is the beginning of a new subslide; that is
+  to say, a new slide, but that `reveal.js` will display *below* the
+  previous one instead of on the right;
+
+* **fragment**: this is to split the contents of one slide into
+  pieces; a cell marked as a fragment will create a break inside the
+  slide; it will not show up right away, you will need to press Space
+  one more time to see it.
+
+* **skip**: this cell will be ignored altogether when presenting the slideshow.
+
+* **notes**: a cell for the presentation mode under reveal (xxx is that a fact?)
```

Comment:
Those are the cells to be rendered in the speaker view, but that is not working yet on master as we discussed in another issue.

Reply:
I made a note on that, but I think I still do not fully understand yet these 2 kinds; if they both are meant to show up in speaker view, what's the difference between them then ?

Reply:
**skip** cells are skipped from the default view and also from the speaker view. In contrast, **notes** cells are "skipped" from the default view but they are visible in the speaker view.